src/components/ActionMenu/menuTemplate.js

```diff
@@ -197,7 +200,7 @@ export const menuTemplate = ({
           key={key}
           caption="ui-data-import.deleteSelectedLogs"
           icon="trash"
-          isDisabled={entity.isDeleteAllLogsDisabled()}
+          isDisabled={entity.isDeleteAllLogsDisabled() || hasNoPermissionToDelete}
```

Comment:
I think it would be better if we could hide `Delete selected logs` button all together instead of disabeling. You can do smth like this:
```js
        <IfPermission perm={permissions.DELETE_LOGS}>
          <Default
            key={key}
            caption="ui-data-import.run"
            icon="play"
            dataAttributes={{ 'data-test-run-item-menu-button': '' }}
            onClick={handleRun}
          />
        </IfPermission>
```

Reply:
Agree, so to be consistent with the View All page.
Also, checkboxes should be suppressed in this case.